documentation/product/auto-discovery/topics/setting-up-data-collectors.md

```diff
@@ -14,7 +14,11 @@ The Data Collector must initiate all communication between the Data Collector an
 
 ## Prerequisites
 
-You must install the Hyperview Data Collector on at least one machine (physical or virtual, running a supported operating system) with network access to your devices. You **cannot** install multiple instances of the Data Collector on the same device or register the same device with more than one Hyperview instance.
+You must install the Hyperview Data Collector on at least one machine (physical or virtual, running a supported operating system) with network access to your devices.
+
+You **cannot** install multiple instances of the Data Collector on the same device or register the same device with more than one Hyperview instance.
+
+Data Collectors must have **unique** names. If you are planning to use the Data Collector with {ref}`Asset Tracker<assettracker-doc>` (RFID asset tracking solution), or as an SNMP trap aggregator, its IP address must remain static.
```

Comment:
It might be better to write it as "AssetTracker".

Reply:
Done, Thank you!